frontend/test/metabase/scenarios/question/nested.cy.spec.js

```diff
@@ -214,4 +214,75 @@ describe("scenarios > question > nested", () => {
       .closest(".List-item")
       .contains("Auto binned");
   });
+
+  it.skip("should apply metrics including filter to the nested question (metabase#12507)", () => {
+    const METRIC_NAME = "Discount Applied";
+
+    withSampleDataset(({ ORDERS }) => {
+      cy.log("**-- 1. Create a metric with a filter --**");
+
+      cy.request("POST", "/api/metric", {
+        name: METRIC_NAME,
+        description: "Discounted orders.",
+        definition: {
+          aggregation: [["count"]],
+          filter: ["not-null", ["field-id", ORDERS.DISCOUNT]],
+          "source-table": 2,
+        },
+        table_id: 2,
+      }).then(({ body: { id: METRIC_ID } }) => {
+        // "capture" the original query because we will need to re-use it later in a nested question as "source-query"
+        const ORIGINAL_QUERY = {
+          aggregation: ["metric", METRIC_ID],
+          breakout: [["binning-strategy", ["field-id", 15], "default"]],
+          "source-table": 2,
+        };
+
+        cy.log(
+          "**-- 2. Create new question which uses previously defined metric --**",
+        );
+
+        cy.request("POST", "/api/card", {
+          name: "Orders with discount applied",
+          dataset_query: {
+            database: 1,
+            query: ORIGINAL_QUERY,
+            type: "query",
+          },
+          display: "bar",
+          visualization_settings: {
+            "graph.dimension": ["TOTAL"],
+            "graph.metrics": ["count"],
```

Comment:
```suggestion
            "graph.metrics": [METRIC_NAME],
```

Reply:
There is no const defined as `METRIC_NAME`. Did you want to say to use metric name here instead of "Count of rows"?